src/sentry/static/sentry/app/views/settings/organization/auditLog/auditLogList.jsx

```diff
@@ -52,21 +52,20 @@ class AuditLogList extends React.Component {
     let {pageLinks, entries, eventType, eventTypes, onEventSelect} = this.props;
     let hasEntries = entries && entries.length > 0;
 
+    let options = [
+      {value: '', label: t('Any action')},
+      ...eventTypes.map(type => ({label: type, value: type})),
+    ];
```

Comment:
the X button is behaving incorrectly here (it should reset value to `''`, but it's setting it to nul which matches no events

Reply:
other than that it's good